### PR TITLE
IAA can be traitor

### PR DESCRIPTION
**Pull request by Birdtalon**
Considering that the role of Brig Physician is being discussed in #8718 I have wondered for a while why pencil pushers are protected from being antagonists.

IAA has a lot more scope for doing interesting things with paperwork and intrigue which is rarely explored currently. The focus on an IAA traitor would be making their forgeries and falsehoods believable enough to sow descent under the noses of the command staff.

Basically I really like the idea of a behind the scenes puppet master sabotaging everything without ever getting his hands dirty.

🆑 
tweak: IAA can now be antags.
/🆑

### CODE REVIEW

**Comment by Ty-Omaha** [comment]
"Let's make roles that are supposed to be trustworthy and reliable untrustworthy and unreliable"

**Comment by FreeStylaLT** [comment]
Lots of IAA work is handled Admin-side, having traitor IAAs make false faxes or claims can only bring interest to the round, while still remaining within full control of most Admins. I don't actually see much wrong with this.

**Comment by Ty-Omaha** [comment]
Not really, most of their work is them investigating, taking complaints, talking with people, getting testimonies and sometimes giving them. And then if nobody on station can fix it then they fax CC.

They're the ones who ensure security is following SOP and proper Space Law, along with the rest of the station. Why NT would want their dedicated investigators that make direct reports to them to be un-mindshielded is beyond me.

**Comment by Decemviri1** [comment]
IAA are already looked down upon with mistrust and suspicion by Security (and other departments) already, however because you can guarantee they're loyal you just put up and shut up (normally). If you add in the possibility that they could be traitors, it'll make their already difficult jobs almost impossible as department's cry that IAA is traitor just to get them gone.

**Comment by Tayyyyyyy** [comment]
Instead of just making all mindshielded roles blanketly antagabble, why not make a new type of antagonist only mindshielded people can get? It'll be sort of like tg's internal affairs, but less violent. Nanotrasen is a kind of scummy corporation that sometimes might want to get rid of someone in its own employ through less than legitimate means. These antagonists would focus on abusing their authority to get their target killed or permad or something. They wouldn't get a syndicate uplink but maybe they could get an NT uplink with stuff more focused around forgery and fabricating false charges or something.

**Comment by Renaultius** [comment]
I feel like the trust afforded to mindshielded personnel is minor compared to normal, anyway. Most people can't even tell if someone is mindshielded and some ways of turning people into antags go through mindshields (slings, for example), so I feel like no one is completely above suspicion. Especially since a lot that the IAA does involves networking (introducing evidence from person 1 and person 2 to the captain, for example), so even if one IAA is a traitor I feel like the chain of command could take the hit relatively painlessly.

The only people who would treat the IAA with additional suspicion would probably be Security or some particularly paranoid command members. And Security never listens to the IAA unless it's convenient to them, so it hardly matters.

The argument that Nanotrasen would be silly not to mindshield them doesn't really hold water, since they fail to mindshield members of frickin' Command (to say nothing of the rest of the crew, I mean maybe mindshielding is expensive or something but it'd really help stop those pesky cultists from summoning dark gods every few shifts). The IAAs even have the NT Rep, a mindshielded member of Command that, though a karma role, is often more active in my experience than the IAA anyway. Considering the second-in-command all-access-wielding HoP can be an antag, I see no reason why the relatively harmless paper-pusher couldn't be.

I also like the idea that the IAA is less Command/Security and more crew; the defense lawyer and the friendly HR officer who actually sticks up for the crewmembers against Big NT. The lack of special shielding reinforces that. But now I'm rambling. :P

**Comment by Ty-Omaha** [comment]
>Considering the second-in-command all-access-wielding HoP can be an antag, I see no reason why the relatively harmless paper-pusher couldn't be.

When the HoP is running around the brig people take note, IAA's always are running around the brig and would go relatively unnoticed.

**Comment by Fox-McCloud** [comment]
IAA doesn't create emergent problems in gameplay that brig physician does.

**Comment by Renaultius** [comment]
> When the HoP is running around the brig people take note, IAA's always are running around the brig and would go relatively unnoticed.

An IAA would still have to go through a couple airlocks to get to anything important, leaving them plenty of time to be discovered. Anyone with basic Security access could do the same (Members of Command, for example), with the only advantage the IAA having being more of a reason to be there. I feel like having the access to walk through any airlock (as HoP) trumps having a better excuse considering I've waltzed into Security plenty of times as HoP, CMO, and RD with hardly a blink of an eye, anyway. After all, it only takes a minute or two to steal some stuff and leave, even faster with full access, so you hardly need to hang around long enough for people to think to ask what the fuck you're doing there in the first place.

IAA traitor, in other words, doesn't strike me as OP at all; the only question in my mind is whether a traitor IAA improves the experience or detracts from it. Since the IAA is a pretty RP-heavy role that requires a lot of time and investment for little payoff in the first place (and considering it's one of the least played roles, if I'm not mistaken), maybe shaking it up isn't a bad idea?

**Comment by EvadableMoxie** [comment]
The IAA should not be traitors for the same reason the NT Rep and Magistrate should not be. They are there to make sure things are run smoothly and correctly. It's hard enough for an IAA to stop power-tripping sec from abusing their power as is.  Muddy the waters by adding the chance that the IAA is not acting in good faith just further reduces their ability to curtail abusive security.

**Comment by DarkPyrolord** [comment]
I'm going to say no to this one, lots of the IAAs role via faxing is handled on the admin side of things and in-game are already given the cold shoulder by most people who play security regularly, most of the time it is minor stuff or we don't have time to respond to faxes.

👎 from me

**Comment by KasparoVy** [comment]
Thank God
